backend/src/test/java/de/jankahle/capstone/service/RecipeTextFilterServiceTest.java

```diff
@@ -0,0 +1,117 @@
+package de.jankahle.capstone.service;
+
+import de.jankahle.capstone.model.Ingredient;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.Optional;
+import java.util.stream.Stream;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+
+class RecipeTextFilterServiceTest {
+
+    private  String testRecipeString = "a WÜURZIGES KARTOFFELGRATIN\n" +
+            "4 ' ZUTATEN\n" +
+            "a — .\n" +
+            "> BD utspin © 4 ©\n" +
+            "m Da -\n" +
+            "Sn ill a m ir ie ie ir 8 (66) Rezept bewerten Personen\n" +
+            "Pa | -\n" +
+            "BZ .\n" +
+            "z a 1kg Kartoffeln, mehligkochend\n" +
+            "SU) EEE“ DD somn POT sim\n" +
+            "1 MW 4 1 Knoblauchzehe\n" +
+            "en i 400 ml Schlagsahne\n" +
+            "Di 4 ©\n" +
+            "z MS A 100 g Parmesan, gerieben\n" +
+            "X Se f { 50 g Butter\n" +
+            "N 1/2 Bund Petersilie\n" +
+            "etwas Salz\n" +
+            "ZUBEREITUNG\n" +
+            "10 Pr. JS Kartoffel Allrounder\n" +
+            "© Den Backofen auf 180 °C (Umluft) vorheizen. Die Kartoffeln schälen und in ca. 0,5 cm feine\n" +
+            "Scheiben schneiden. Den Knoblauch ebenfalls schälen und halbieren.\n" +
+            "© Die Sahne mit der Creme Fraiche, dem Knoblauch, dem Kartoffel Allrounder und 3 Prisen Salz in\n" +
+            "einem Topf aufkochen lassen und vom Herd nehmen. Den Knoblauch herausnehmen und 50 g\n" +
+            "Parmesan unterheben.\n" +
+            "© Eine Auflaufform gleichmäßig mit der Butter einfetten und die Kartoffelscheiben darin leicht\n" +
+            "überlappend aufschichten, mit der Sahne-Mischung übergießen und mit dem übrigen Parmesan\n" +
+            "bestreuen.\n" +
+            "0 Den Gratin ca. 50 Minuten im Ofen garen, bis es leicht gebräunt ist. Die Petersilie waschen,\n" +
+            "trocken tupfen und fein schneiden. Den Kartoffelgratin mit der Petersilie bestreut servieren.\n";
+
+    @Test
+    void parseStringToRecipeTest() {
+//TODO: write Recipe test
```

Comment:
where is your test 🥺

Reply:
As mentioned above, this is beyond the scope of this PR